superset-frontend/cypress-base/cypress/integration/chart_list/card_view.test.ts

```diff
@@ -107,11 +113,12 @@ describe('chart card view', () => {
     cy.get('[data-test="modal-cancel-button"]').click();
   });
 
-  it('should edit correctly', () => {
+  // flaky
+  xit('should edit correctly', () => {
```

Comment:
Why did we nix  so many tests? Were they causing problems in other PRs, too? If yes, I think it would be better if they were turned off in a separate PR so we can keep  better track of them.

Reply:
All of these tests failed and passed at various times, as well as on other PRs. We did a little investigation but haven't found the cause of flakiness yet.